.semaphore/semaphore.yml

```diff
@@ -53,6 +53,9 @@ blocks:
         - name: nodejs/next-js
           commands:
             - "rake app=nodejs/next-js app:test"
+        - name: ruby/jruby-rails6
+          commands:
+            - "rake app=ruby/padrino app:test"
```

Comment:
```suggestion
            - "rake app=jruby/rails6 app:test"
```

Reply:
@luismiramirez can you update this line so that it runs the right tests?

Reply:
Fixed here https://github.com/appsignal/test-setups/pull/58